src/java/org/apache/cassandra/config/DataRateSpec.java

```diff
@@ -228,6 +217,99 @@ public String toString()
         return Math.round(quantity) + unit.symbol;
     }
 
+    /**
+     * Represents a data rate used for Cassandra configuration. The bound is [0; Long.MAX_VALUE) in bytes per second.
+     * If the user sets a different unit - we still validate that converted to bytes per second the quantity will not exceed
```

Comment:
```suggestion
     * If the user sets a different unit, we still validate that converted to bytes per second the quantity will not exceed
```
Applies to a few other classes as well...